platforms/hyperledger-besu/charts/node_tessera/templates/deployment.yaml

```diff
@@ -0,0 +1,261 @@
+          var=$(cat /secret/keys/password)         
+          cat << EOF > /docker-entrypoint-initdb.d/createTables.sql
+          UPDATE mysql.user SET authentication_string = PASSWORD('$var') WHERE User = 'demouser';
+          UPDATE mysql.user SET authentication_string = PASSWORD('$var') WHERE User = 'root';
+          CREATE TABLE IF NOT EXISTS ENCRYPTED_TRANSACTION (ENCODED_PAYLOAD BLOB NOT NULL, HASH VARBINARY(100) NOT NULL, TIMESTAMP BIGINT, PRIMARY KEY (HASH));
+          CREATE TABLE IF NOT EXISTS ENCRYPTED_RAW_TRANSACTION (ENCRYPTED_KEY BLOB NOT NULL, ENCRYPTED_PAYLOAD BLOB NOT NULL, NONCE BLOB NOT NULL, SENDER BLOB NOT NULL, TIMESTAMP BIGINT, HASH VARBINARY(100) NOT NULL, PRIMARY KEY (HASH));
+          CREATE TABLE IF NOT EXISTS PRIVACY_GROUP(ID VARBINARY(100) NOT NULL, LOOKUP_ID BLOB NOT NULL, DATA BLOB NOT NULL, TIMESTAMP BIGINT, PRIMARY KEY (ID));
+          EOF
+        volumeMounts:
+        - name: mysql
+          mountPath: /docker-entrypoint-initdb.d
+        - name: certificates
+          mountPath: /secret      
+      containers:
+      - name: mysql-db
+        image: {{ .Values.images.mysql }}
+        imagePullPolicy: Always
+        env:
+        - name: "MYSQL_ROOT_PASSWORD"
+          value: ""
+        - name: MYSQL_DATABASE
+          value: "{{ .Values.tessera.dbname }}"
+        - name: MYSQL_USER
+          value: "{{ .Values.tessera.dbusername }}"
+        - name: MYSQL_PASSWORD
+          value: /secret/keys/password
+        ports:
+          - containerPort: {{ .Values.tessera.ports.db }}
+        volumeMounts:
+        - name: mysql
+          mountPath: /docker-entrypoint-initdb.d
+        - name: certificates
+          mountPath: /secret
+        - name: {{ .Values.tessera.name }}-mysql
+          mountPath: "/var/lib/mysql"
+          subPath: mysql
+      - name: tessera
+        image: {{ .Values.images.tessera }}
+        command: ["sh"]
+        args:
+        - "-cx"
+        - "chmod 600 $TESSERA_HOME/crypto/keys/tm.key;
+          mkdir -p $TESSERA_HOME/logs;
+          mkdir -p $TESSERA_HOME/tm;
+          DDIR=$TESSERA_HOME/tm;
+          PASSWORD=$(cat $TESSERA_HOME/crypto/keys/password);
+
+          printenv;
+          wget -q http://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-8.0.15.tar.gz;
+          tar -xf mysql-connector-java-8.0.15.tar.gz;
+          cp /mysql-connector-java-8.0.15/mysql-connector-java-8.0.15.jar $TESSERA_HOME;
+          SQL_JAR=$TESSERA_HOME/mysql-connector-java-8.0.15.jar;
+
+          TESSERA_JAR=/tessera/tessera-app.jar;
+          CONFIG_TMPL=$(cat ${TESSERA_HOME}/tessera-config.json.tmpl);
+
+          echo $CONFIG_TMPL >  ${DDIR}/tessera-config-with-hosts.json;
+          cat  ${DDIR}/tessera-config-with-hosts.json;
+
+          TESSERA_MAIN=com.quorum.tessera.launcher.Main;
+          java -Xms128M -Xmx128M -cp $SQL_JAR:$TESSERA_JAR:. $TESSERA_MAIN -configfile ${DDIR}/tessera-config-with-hosts.json --jdbc.password $PASSWORD;
```

Comment:
For tessera 21.4.0; overriding the config didn't work for me; had to use like -o jdbc.password=$PASSWORD. Does this work for 21.1.1 ?